### PR TITLE
removed all travis links and replaced it with github actions

### PR DESCRIPTION
**Pull request by fabianpage**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,48 +22,54 @@ fountain data from Open Street Map, Wikidata, Wikimedia Commons, and Wikipedia.
...
-    - run `> npm run sync_datablue` to replicate the two constants files from datablue (the datablue server must be running)  
-    - Launch the server by running ``~/git/proximap$ npm run start 2>&1 | tee npm_pm_`date +%y%m%d_%H%M%S`.log``. You can view the application by navigating to <a href="http://localhost:4200" target="_blank" >http://localhost:4200</a>. The app will automatically reload if you change any of the source files. 
+   - Install required node packages by running `> npm install`. If you update the project with `> git pull`, be sure to run npm `> npm install` again to update all packages.
+   - run `> npm run sync_datablue` to replicate the two constants files from datablue (the datablue server must be running)
+   - Launch the server by running `` ~/git/proximap$ npm run start 2>&1 | tee npm_pm_`date +%y%m%d_%H%M%S`.log ``. You can view the application by navigating to <a href="http://localhost:4200" target="_blank" >http://localhost:4200</a>. The app will automatically reload if you change any of the source files.
```
Comment (robstoll) [inline README.md]: remove space again? or was this on purpose?
```suggestion
   - Launch the server by running ``~/git/proximap$ npm run start 2>&1 | tee npm_pm_`date +%y%m%d_%H%M%S`.log``. You can view the application by navigating to <a href="http://localhost:4200" target="_blank" >http://localhost:4200</a>. The app will automatically reload if you change any of the source files.

```

File: git.version.js [hunk 2]
```diff
@@ -14,8 +14,7 @@ const exec = util.promisify(require('child_process').exec);
 
 async function createVersionsFile(filename) {
   const revision = (await exec('git rev-parse --short HEAD')).stdout.toString().trim();
-  // use travis branch environment variable if available, because in Travis the git branch is called HEAD by default
-  const branch = process.env.TRAVIS_BRANCH || (await exec('git rev-parse --abbrev-ref HEAD')).stdout.toString().trim();
+  const branch = (await exec('git rev-parse --abbrev-ref HEAD')).stdout.toString().trim();
```
Comment (robstoll) [inline git.version.js:17]: could be we still have `process.env.TRAVIS_BRANCH || ` in datablue, you could remove it there as well

**Review by robstoll — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (fabianpage) [reply README.md:48]: was the autoformater. I would keep it, this will be adated anyway when we introduce prettier.

File: git.version.js (hunk 2 above)
Comment (fabianpage) [reply git.version.js:17]: In datablue i din't found travis anymore.

File: README.md (hunk 1 above)
Comment (fabianpage) [reply README.md]: ah inside the backticks. I will remove that.

**Pull request merged**